hooks/rnc-format-validate.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+import subprocess
+from pathlib import Path
+
+
+def main():
+
+    proc = subprocess.Popen(
+        ["git", "diff", "--name-only", "--cached"], stdout=subprocess.PIPE
+    )
+    changed_files = proc.stdout.readlines()
+    changed_files = [f.decode("utf-8") for f in changed_files]
+    changed_files = [f.strip() for f in changed_files]
```

Comment:
You can probably achieve the same thing with:

```suggestion
    cmd = ["git", "diff", "--name-only", "--cached"]
    changed_files = subprocess.check_output(cmd, text=True).splitlines()
```